### ISSUE TITLE
findOrCreate does not seem to support arrays

### ISSUE THREAD

**Issue by gotmikhail** [balderdashy/sails-docs#432]
Sails documentation http://sailsjs.org/#!/documentation/reference/waterline/models/findOrCreate.html lists arrays of objects, `[{}]`, as valid for `Find criteria` and `Records to create`.

However, sending an array results in this error:

`Unknown column 'user.[object Object]' in 'where clause'`

Searching stackoverflow, google groups, and code, it doesn't look like it actually supports `arrays`. Is documentation wrong?

**Comment by sailsbot** [comment]
Thanks for posting, @gotmikhail.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
